homeassistant/components/webostv/media_player.py

```diff
@@ -313,16 +325,22 @@ def media_image_url(self) -> str | None:
     @property
     def supported_features(self) -> int:
         """Flag media player features that are supported."""
+        if self.state == STATE_OFF and self._supported_features is not None:
+            return self._supported_features
+
         supported = SUPPORT_WEBOSTV
```

Comment:
```suggestion
        self._supported_features = SUPPORT_WEBOSTV
```
You can store the result directly in `self._supported_features`

Reply:
I don't think I can do this change because we don't want to set this when the TV is off!

If the TV is off and we set that value here, it will skip the restore as I am checking there `self._supported_features is None`

Reply:
Ok, well for now I can't switch to the files tab, so I'll ask @MartinHjelmare to look at it.
![image](https://user-images.githubusercontent.com/1858925/152768909-00056dc0-4d2a-4e7c-8fc6-962b1ea65209.png)

I am ok with leaving it like this for now, it can be improved in a future PR.